addons/mass_mailing/models/mailing.py

```diff
@@ -853,6 +861,7 @@ def action_select_as_winner(self):
             raise ValueError(_("A/B test option has not been enabled"))
         final_mailing = self.copy({
             'ab_testing_pc': 100,
+            'name': _('%(subject)s (final)', subject=final_mailing.name), # Add suffix on name to show it's the final mailing
```

Comment:
```suggestion
            'name': _('%(subject)s (final)', subject=self.name), # Add suffix on name to show it's the final mailing
```

Reply:
I added the final_mailing by mistake and I have tried using the solution given by you but it is not working. it only assigns the subject(self.name) and ignores any string afterward,

it has a strange behaviour that if we add anything before ` %(ex)s` it will be added but after it, anything we add will not be added 

https://i.imgur.com/rbXtsT4.png

Reply:
Mailing inherits from the 'utm.source.mixin' that overrides the create to handle the name. Maybe it is worth checking the whole flow (check what happens in ``copy_data``, ``copy``, and in the ``source.create()`` method). That's a good exercice to try to find the source of this :)

Reply:
Hello @tde-banana-odoo 

You were right about `utm.source.mixin`, as in copy method of it https://github.com/odoo/odoo/blob/c025a8ad8496570f4ef78f539df901c887788f9d/addons/utm/models/utm_source.py#L85 line changes the `default['name']` based on value of `self.name`. and that is the reason that the name again changes.

We can also change it to 
`default['name'] = self.env['utm.mixin']._get_unique_names(self._name, [default['name'] or self.name])[0]`
or we can go with the current solution.

Again thanks for guiding me to the right path :)

Reply:
I think we can fix the mixin directly :)